Source/Core/buildModuleUrl.js

```diff
@@ -87,12 +90,15 @@ define([
             }
         }
 
+        var url = implementation(moduleID);
```

Comment:
@mramato @shunter what I'm seeing is that `implementation` in built Cesium is `buildModuleUrlFromBaseUrl` on the main process but `buildModuleUrlFromRequireToUrl` on the worker, but for the worker this returns an absolute URL for non-built and a relative one for built?

Reply:
This is easiest to reproduce using the `Polyline` Sandcastle.

Reply:
One last observation, this line does something different between built/unbuilt:
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/TaskProcessor.js#L151

And then this https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Workers/cesiumWorkerBootstrapper.js#L227 gets set differently.